shed/utils.py

```diff
@@ -23,7 +25,12 @@ def to_event_model(data, output_info):
     -----
     This is only for demonstration/example use, do not use for production.
     """
-    es = EventStream(md={'source': 'to_event_model'}, output_info=output_info)
+    if md is None:
```

Comment:
why not have `md={}` in default `kwarg` behaviour? im indifferent to either, just asking

Reply:
Mutables in kwargs are bad since the function can drift if the mutable is mutated
http://reinout.vanrees.org/weblog/2012/04/18/default-parameters.html

Reply:
with that logic, should we not then also do 
```python
else:
    md = md.copy()
```

Reply:
we could do that, in case you don't want our internal metadata appended to your original metadata.

Reply:
Although when I made a test for this it didn't fail so there may be something more happening.